osf/metrics.py

```diff
@@ -325,3 +325,35 @@ def get_latest_institution_project_document(cls, institution):
         response = search.execute()
         if response:
             return response[0]
+
+
+class RegistriesModerationMetrics(MetricMixin, metrics.Metric):
+    registration_id = metrics.Keyword(index=True, doc_values=True, required=True)
+    provider_id = metrics.Keyword(index=True, doc_values=True, required=True)
+    trigger = metrics.Keyword(index=True, doc_values=True, required=True)
+    from_state = metrics.Keyword(index=True, doc_values=True, required=True)
+    to_state = metrics.Keyword(index=True, doc_values=True, required=True)
+    user_id = metrics.Keyword(index=True, doc_values=True, required=True)
+    comment = metrics.Keyword(index=True, doc_values=True)
```

Comment:
Do we need to have `doc_values=True` set on comment? I can't think of a case where the comment would need to be aggregated?

Reply:
I think your right, it's not necessary, changed!